packages/core/src/lib/prepareParams.ts

```diff
@@ -9,6 +9,8 @@ import stream from 'node:stream';
 import caseless from 'caseless';
 import DatauriParser from 'datauri/parser.js';
 import datauri from 'datauri/sync.js';
+// `get-stream` is included in our bundle, see `tsup.config.ts`
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Annoying that `import/no-extraneous-dependencies` doesn't let us supply a list of packages to ignore.

Reply:
agreed... i tried playing around with our ESLint config and got nowhere